### PR TITLE
Chore/validate snapshot test

### PR DESCRIPTION
**Pull request by germfy**
Automatized testing in Ci/CD Workflow.

### CODE REVIEW

File: .github/workflows/dev_branch_linting_testing.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      - name: checkout DowJones Factiva Core repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'wizeline/factiva-core-python'
+          ref: 'development'
+          path: 'main'
+
+      - name: Directory Listing & copy directories
+        run: |
+          cp -R ./main/src/factiva/core ./main/src/factiva/helper ./src/factiva/
```
Comment (vic3r) [inline .github/workflows/dev_branch_linting_testing.yml:79]: 🙈

File: .github/workflows/master_branch_linting_testing_build.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      - name: checkout DowJones Factiva Core repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'wizeline/factiva-core-python'
+          ref: 'development'
+          path: 'main'
+
+      - name: Directory Listing & copy directories
+        run: |
+          cp -R ./main/src/factiva/core ./main/src/factiva/helper ./src/factiva/
```
Comment (vic3r) [inline .github/workflows/master_branch_linting_testing_build.yml:78]: 🙈

**Review by vic3r — APPROVED** [review]
👍

**Review by cerritows — APPROVED** [review]
👯

**Pull request merged**